File: HaberUygulamaJetpack/app/src/main/java/com/example/haberuygulamajetpack/model/HaberModel.kt
package com.example.haberuygulamajetpack.model

import java.io.Serializable

data class HaberModel(
    val id: Int,
    val baslik: String,
    val icerik: String,
    val media_type: String,
    val media_url: String,
    val yayinlanma_tarihi: String,
    val sondakika: Int,
    val yazar_id: Int,
    val tur_id: Int,
    val ad: String,
    val soyad: String,
    val unvan: String,
    val tur_adi: String
) : Serializable

File: haber_uygulama_react_native/src/model/haber_model.ts
export type HaberModel = {
    id: number;
    baslik: string;
    icerik: string;
    media_type: string;   // "video" | "image" vs
    media_url: string;
    yayinlanma_tarihi: string;
    sondakika: number;
    yazar_id: number;
    tur_id: number;
    ad: string;
    soyad: string;
    unvan: string;
    tur_adi: string;
};
